misc/tcb-auth.sysusers

```diff
@@ -0,0 +1,4 @@
+# Additional access group for the tcb password shadowing scheme
+# when "TCB_AUTH_GROUP yes" is in use.  the "nobody" user is
```

Comment:
This comment still says `nobody`. It should probably say `root` there, and `the` should be capitalized.

Reply:
Fixed in rebased commit.